### DISCUSSION TITLE
Codespace failing to connect

### DISCUSSION THREAD

**Discussion by oofpouf** [community/community#26707] [Codespaces]
<p>I’ve tried to access codespace via internet browser, but am unable to connect and have been stuck at this page in the attached image for hours. I have even tried to download visual studio code and the github extension, but am unable to do so. The notification in visual code studio shows that it is still “setting up remote connection” for hours as well.</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Welcome <a class="mention" href="/oofpouf">@oofpouf</a>!</p>
<p>I see that you recently posted a topic about Codespaces Beta. I wanted to introduce you to a recent change that you may not have yet seen: <a href="https://github.community/t/codespaces-beta-expands-to-discussions/165089">Codespaces Beta Expands to Discussions </a></p>
<p>Please take a look for the new public location in order to continue this conversation with our team. Thank you again for being here!</p>
<blockquote>
<p>The Codespaces Beta category has moved over to Discussions! Our Codespaces team is eager to continue in conversations and knowledge-sharing with you as you share your questions and feedback with us. Please head over to the new public space for <a href="https://github.com/github/feedback/discussions/categories/codespaces-feedback">Codespaces Feedback</a> to take a look!</p>
<h4>Resources on Discussions to Get Started</h4>
<ul>
<li><a href="https://docs.github.com/en/discussions">GitHub Discussions Beta</a></li>
<li><a href="https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion">Participating in a discussion</a></li>
</ul>
</blockquote>
